### PR TITLE
dpn-242_bug fix

### PR DESCRIPTION
**Pull request by smutniak**
fixing dpn-242 bug by changing default datetime in db/schema.rb

### CODE REVIEW

**Comment by malakai97** [comment]
There are a couple of things going on here.
1. The db/schema.rb is generated by rails, primarily by running rake db:migrate.  So this change is likely to be overwritten.
2. This file will be different depending on which database adapter (sqlite, mysql, postgres, etc) ActiveRecord has been configured to use.  This is despite rails devs claiming schema.rb is agnostic.  The version committed in the repo is for sqlite.  You'll notice that it doesn't have an foreign keys, despite the migrations creating them.  This is because sqlite doesn't have foreign key support.  

One we we could go about finding these types of errors before they hit production is configuring travis to build the application once with sqlite, once with mysql, etc.

It may be that the best way to fix this is to remove db/schema.rb from the repo.  That would force users to generate their own with the database adapter they've configured.  However, there's little way to guarantee that a correct schema.rb is generated--it seems like we'd be asking for trouble.  

tl;dr: This fix isn't going to address the underlying issue, and I'm not aware of a good strategy for addressing the underlying issue at this time.

**Comment by malakai97** [comment]
Closing this as this isn't a bug.  See my previous comment.

**Comment by smutniak** [comment]
Makes sense.  I'll remove the dpn-242_bug branch.
